src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/TotrinnskontrollService.kt

```diff
@@ -27,35 +33,50 @@ class TotrinnskontrollService(
     private val behandlingshistorikkService: BehandlingshistorikkService,
     private val behandlingService: BehandlingService,
     private val tilgangService: TilgangService,
+    private val totrinnskontrollRepository: TotrinnskontrollRepository,
 ) {
 
     /**
      * Lagrer data om besluttning av totrinnskontroll
      * og returnerer navIdent til saksbehandleren som sendte behandling til beslutter
      */
+
     @Transactional
-    fun lagreTotrinnskontrollOgReturnerBehandler(
+    fun opprettTotrinnskontroll(saksbehandling: Saksbehandling, beslutteVedtak: BeslutteVedtakDto): Totrinnskontroll {
+        val nytotrinnskontroll = Totrinnskontroll(
+            behandlingId = saksbehandling.id,
+            sporbar = Sporbar(),
+            saksbehandler = Sporbar().opprettetAv,
```

Comment:
Hvis denne skal settes hadde jeg nok gjort det sånn
```suggestion
            saksbehandler = SikkerhetContext.hentSaksbehandlerEllerSystembruker()
```